pkg/rehearse/jobs.go

```diff
@@ -20,34 +28,30 @@ import (
 	"k8s.io/client-go/rest"
 )
 
-type prowJobClientWithDry struct {
-	pj.ProwJobInterface
-
-	dry bool
-}
+const (
+	rehearseLabel = "ci.openshift.org/rehearse"
+)
 
 // NewProwJobClient creates a ProwJob client with a dry run capability
 func NewProwJobClient(clusterConfig *rest.Config, namespace string, dry bool) (pj.ProwJobInterface, error) {
+	if dry {
+		pjcset := pjclientsetfake.NewSimpleClientset()
+		pjcset.Fake.PrependReactor("create", "prowjobs", func(action testing.Action) (bool, runtime.Object, error) {
+			pj := action.(testing.CreateAction).GetObject().(*pjapi.ProwJob)
+			jobAsYAML, err := yaml.Marshal(pj)
+			if err != nil {
+				return true, nil, fmt.Errorf("failed to marshal job to YAML: %v", err)
+			}
+			fmt.Printf("%s\n", jobAsYAML[:10])
```

Comment:
What's in those ten characters?

Reply:
That's embarrassing… It was annoying to get two screens of yaml thrown at me every time I tested it =)

Removed.